docs/app/pages/css/css-sections/icons/ux-icons/ux-icons.component.less

```diff
@@ -0,0 +1,21 @@
+.icon-search-items-container {
+    display: flex;
+    align-items: center;
+    padding: 5px 0;
+
+    .icon-selector-text {
+        justify-content: center;
+        flex-direction: column;
```

Comment:
These don't do anything without `display: flex` and therefore should be removed.

Also if you add `flex: none` in here you can fix the nasty line breaking issue.